### DISCUSSION TITLE
SEB and screen recording with Camtasia: screen flickering

### DISCUSSION THREAD

**Discussion by AntonT76** [SafeExamBrowser/seb-win-refactoring#449] [Q&A] [answered]
Dear community,

I try to use Camtasia to record a screen together with using SEB. After setting permitted/prohibited processes in SEB correctly I startet a recording, open SEB and start the exam. After that I closed SEB and stopped recording. It seems to work fine.

But if I open the screen recording file the video only shows a screen flickering with starting window of SEB (screen flickers between "initializing configuration" and "validating remote session policy") and no further content of the screen which I have seen during the recording. FYI: Flickering started by opening the SEB and ends with closing SEB.

I'm using Camtasia 2019, SEB 3.3.2 and Win 10. Attached you can find a screenshot.
![SEB_Camtasia_flickering](https://user-images.githubusercontent.com/66412602/185585353-613e67e4-9cac-4a92-b5b5-27614263e66a.png)

Any ideas whats wrong?

thanks, br Anton

**Comment by dbuechel** [comment, marked as answer]
There are quite a few screen recording applications which unfortunately only record the default Windows desktop. Thus, if you're using kiosk mode Create New Desktop, all the content of the main SEB user interface (Browser etc.) won't be recorded. A workaround would be to use kiosk mode Disable Explorer Shell (see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane) or then report the issue to the software manufacturer (i.e. a screen recording software should detect a desktop switch and then record the currently active desktop).

**Reply by AntonT76** [reply]
thanks for your answer - the workaround works. br Anton
